custom_model_runner/datarobot_drum/drum/gpu_predictors/nemo_predictor.py

```diff
@@ -0,0 +1,239 @@
+"""
+Copyright 2024 DataRobot, Inc. and its affiliates.
+All rights reserved.
+This is proprietary source code of DataRobot, Inc. and its affiliates.
+Released under the terms of DataRobot Tool and Utility Agreement.
+"""
+import csv
+import io
+import logging
+import os
+import signal
+import subprocess
+import atexit
+
+import numpy as np
+from openai import OpenAI
```

Comment:
How do we ensure that OpenAI is installed? Do we need to add an optional dependency to the drum package?

Reply:
currently it's not supposed to be useable without the drop-in environment.

>  Do we need to add an optional dependency to the drum package

yes

Reply:
I'm not adding this dependency into DRUM for now. It may be useful only for local dev work, when you have DRUM on local machine and NIM server on remote.